environments/sre/main.jsonnet

```diff
@@ -0,0 +1,60 @@
+local sm =
+  (import 'kube-thanos/kube-thanos-servicemonitors.libsonnet') +
+  {
+    thanos+:: {
+      querier+: {
+        serviceMonitor+: {
+          metadata: {
+            name: 'observatorium-thanos-querier',
+            labels: { prometheus: 'app-sre' },
+          },
+          spec+: {
+            selector+: {
```

Comment:
are we intentionally _extending_ the selector but _overriding_ the namespaceSelector?

Reply:
We do. Is this a mistake? I just wanted to add `namespaceSelector`

Reply:
no not necessarily a mistake, just wanted to know if it was intentional and why the reasoning is